libs/server/Auth/AuthenticationSettings.cs

```diff
@@ -116,6 +120,7 @@ public class AadAuthenticationSettings : IAuthenticationSettings
         private readonly IReadOnlyCollection<string> _audiences;
         private readonly IReadOnlyCollection<string> _issuers;
         private IssuerSigningTokenProvider _signingTokenProvider;
+        private bool _validateUsername;
         private bool _disposed;
 
```

Comment:
This is not the scope of this PR but this file is getting too big. We might need to refactor it into smaller files. If you are up to it please separate the AuthenticationSettings under Settings/ folder and then for each class have a separate file.